### DISCUSSION TITLE
Seperate Local and Production Environmental Variables

### DISCUSSION THREAD

**Discussion by ephraimduncan** [prisma/prisma#15458] [Q&A] [answered]
## Problem

When I am developing locally. I have a specific url for a local database. When I host the db online, I get a different connection url for the database. When I am working, I have to switch to the local url and when I am deploying changes or anything it, I have to switch back to the production url and these are confusing at times, sometimes you forget to switch and everything breaks.

## Suggested solution

An option in the Prisma Schema to specifically set the local and production database connection url to avoid any mix. The local env runs locally and the production env runs in production depending on the NODE_ENV value

## Alternatives

A Github Action that check your Prisma Schema every time you push a new commit to see if the database url matches the pattern of a local database and the action changes it to a specified production url.

**Comment by millsp** [comment]
Hey @dephraiim is this what you are looking for https://www.prisma.io/docs/guides/development-environment/environment-variables#example-set-the-database_url-environment-variable-in-an-env-file

**Reply by vladi-stevanovic** [reply]
👋 Hello @dephraiim - did you perhaps have a chance to check Pierre's recommendation? Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved!

**Comment by TeePlunder** [comment, marked as answer]
@dephraiim I don't know if @millsp reply helped you but if not I think this can help you: https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
